cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -194,8 +195,11 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run external license
             throw ProgramResult(1)
         }
 
-        if (scanResults.hasIssues) {
-            println("The scan result contains issues.")
+        val hasIssues = scanResults.scanResults.any { (_, results) ->
+            results.any { it.summary.issues.any { issue -> issue.severity != Severity.HINT } }
```

Comment:
This triggered the idea for https://github.com/oss-review-toolkit/ort/pull/3956 on my side so you can do `issue.severity > Severity.HINT` afterwards.